src/Operation/FindAndModify.php

```diff
@@ -264,7 +264,14 @@ public function execute(Server $server)
      */
     public function getCommandDocument(Server $server)
     {
-        return $this->createCommandDocument();
+        $cmd = $this->createCommandDocument();
+
+        // Read concern can change the query plan
```

Comment:
[findAndModify](https://www.mongodb.com/docs/manual/reference/command/findAndModify/) doesn't support a `readConcern` option. This should be removed.

Easy mistake to make as PHPLIB-802 was originally focused on findAndModify.